LICENSE

```diff
@@ -295,6 +295,10 @@ Apache Geode bundles the following files under the MIT license:
   - Timeago v1.3.0 (http://timeago.yarp.com), Copyright (c) 2008-2015 Ryan
     McGeary
   - zTree v3.5.02 (http://zTree.me/), Copyright (c) 2010 Hunter.z
+  - Animal Sniffer v1.17 (http://www.mojohaus.org/animal-sniffer), Copyright
```

Comment:
Shouldn't this Animal Sniffer copyright live in the geode-web/src/main/webapp/META-INF/NOTICE or ./geode-web-management/src/main/webapp/META-INF/NOTICE instead of in the top LICENSE file? 
Generally I don't know when one should be used over the other except in this case the use of Animal Sniffer is only in the geode-web and geode-web-management subprojects so this probably belongs there.